charts/cloudzero-agent/README.md

```diff
@@ -17,255 +17,36 @@ For the latest release, see [Releases](https://github.com/Cloudzero/cloudzero-ch
 
 To use the chart or a beta version, you must add the repository to Helm. Refer to the [`helm repo`](https://helm.sh/docs/helm/helm_repo/) documentation for command details.
 
-#### 1. Add the Helm Chart Repository
+#### 1. Install the Helm Chart
 
 ```console
 helm repo add cloudzero https://cloudzero.github.io/cloudzero-charts
+helm install cloudzero cloudzero/cloudzero-agent \
+    --set apiKey=<YOUR_CLOUDZERO_API_KEY>\
+    --set clusterName=<YOUR_CLUSTER_NAME>
 ```
-> Note: If you intend to use a beta version, refer to the [beta installation document](./BETA-INSTALLATION.md) for the appropriate channel.
 
-#### 2. Add Repository Dependencies
+### Advanced Install (Usually when you have specific security requirements.)
 
-**Cert Manager**
-```console
-helm repo add cert-manager https://charts.jetstack.io    
-```
-
-**Metrics Exporter**
-```console
-helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
-```
-
-#### 3. Update the Helm Repositories
-
-Ensure that the most recent chart versions are available:
-
-```console
-helm repo update
-```
-
-### Install Helm Chart
-
-The chart can be installed directly with Helm or any other common Kubernetes deployment tools.
-
-If installing with Helm directly, execute the following steps:
-
-1. Ensure that the most recent chart version is available:
-```console
-helm repo update
-```
-
-2. Provision a TLS certificate; by default, this chart deploys a `ValidatingWebhookConfiguration` resource, which requires a certificate in order validate requests to the webhook server. See related Kubernetes documentation [here](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#configure-admission-webhooks-on-the-fly).
-
-There are several options for provisioning this certificate. The default is to use a third party tool, [cert-manager](https://cert-manager.io/). If you would prefer not to use cert-manager, see the [Certificate Management](#certificate-management) section for other options.
-
-To use `cert-manager` for certificate management, first install the `cert-manager` CRDs with the following:
-```console
-helm install <RELEASE_NAME> cloudzero/cloudzero-agent \
-    --set insightsController.webhook.issuer.enabled=false \
-    --set insightsController.webhook.certificate.enabled=false \
-    --set insightsController.cert-manager.installCRDs=true
-```
-Alternatively, [install the cert-manager CRDs directly](https://cert-manager.io/docs/installation/helm/) with:
+#### 1. Create and Configure a Values File
 
-```console
-kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.16.2/cert-manager.yaml
-```
-
-3. Fill out all required fields in the `configuration.example.yaml` file in this directory. Rename the file as necessary. Below is an example of a completed configuration file:
 ```yaml
-# -- Account ID of the account the cluster is running in. This must be a string - even if it is a number in your system.
```

Comment:
How could we get this down to 1 single key for the customer instead of asking for region, accountid, and keyid? Ideally we just ask for a singular api key

Reply:
we have a ticket for this here https://cloudzero.atlassian.net/browse/CP-23666